Stratis.Bitcoin.Features.BlockStore.Tests/LoopTests/BlockStoreLoopStepBaseTest.cs

```diff
@@ -60,11 +60,13 @@ internal List<Block> CreateBlocks(int amount)
             return blocks;
         }
 
-        internal Block CreateBlock(int blockNumber)
+        internal Block CreateBlock(int blockNumber, bool bigBlocks = false)
         {
             var block = new Block();
 
-            for (int j = 0; j < 10; j++)
+            var transactions = bigBlocks ? 1000 : 10;
```

Comment:
`int transactionCount` would be better. Also why to replace int with var?

Reply:
Fixed